R/print.R

```diff
@@ -442,6 +442,66 @@ print.bbi_nmboot_summary <- function(x, .digits = 3, .nrow = 10, ...) {
   }
 }
 
+
+#' @describeIn print_bbi Prints the `NM-TRAN` evaluation of a `bbi_nonmem_model`
+#'  object
+#' @export
+print.nmtran_process <- function(x, ...){
+
+  is_valid_print <- function(.x) {
+    if (!is.null(.x)) {
+      length(.x) != 0
+    } else {
+      FALSE
+    }
+  }
+
+  heading <- cli_h1
+  subheading <- cli_h2
+  bullet_list <- cat_bullet
+
+  status <- x[["status"]]
+  if (x[["status_val"]] == 0) {
+    status <- col_green(status)
+  } else {
+    status <- col_red(status)
+  }
+
+  nm_version <- x[["nonmem_version"]]
+  if(is.null(nm_version)) nm_version <- "unknown"
+
+  heading('Status')
+  subheading(status)
+
+  heading("Absolute Model Path")
+  bullet_list(x[[ABS_MOD_PATH]])
+
+  heading("NMTRAN Specifications")
+  cli::cli_bullets(
+    c(
+      "*" = paste0("NMTRAN Executable:  {.path ", x[["nmtran_exe"]], "}"),
+      "*" = paste0("NONMEM Version: {.val ", nm_version, "}"),
+      "*" = paste0("Run Directory: {.path ", x[["run_dir"]], "}")
+    )
+  )
+
+  if (is_valid_print(x[["output_lines"]])) {
+    heading('Output')
+    cat_line(style_italic(x[["output_lines"]]))
+  }
+
+  if (is_valid_print(x[["error_lines"]])) {
+    heading('Additional Errors')
```

Comment:
"Additional Errors" is confusing to me because the previous section with standard output doesn't necessarily have any errors.

*If* you're going to keep standard output and standard error as separate streams, it'd probably be clearest to name these something like "Standard Output" and "Standard Error". Yes, some bbr users may not be familiar with those terms, but calling them anything else is making strong assumptions about what NMTRAN is writing to which stream. For example, "Output" and "Errors" would be confusing because NMTRAN writes messages about control stream errors to standard output.

But my suggestion would be to adjust the processx call so that stderr is sent to stdout. Then just show those combined lines in one section ("Output").

Reply:
Yeah good catch on revisiting this. Wasnt the biggest fan of that heading and was hoping to revisit it during review. Wasn't aware the second option was a possibility, but I agree that sounds like the best approach. I have it implemented now and will push it with my next round of commits.